packages/vscode-extension/src/debug/teamsfxTaskHandler.ts

```diff
@@ -93,6 +93,15 @@ function isNpmInstallTask(task: vscode.Task): boolean {
   return false;
 }
 
+function isDevProxyTask(task: vscode.Task): boolean {
+  if (task) {
+    const name = task.name.trim();
```

Comment:
Does this new task have any specific identifier like `type: "teamsfx"` or `teamsfx: ` prefix?

Reply:
The new tasks are shell commands which are used to check and start dev proxy process in some samples.
![image](https://github.com/OfficeDev/teams-toolkit/assets/16380704/0e5ebd34-e362-4cd3-9178-89c342b8a425)

Reply:
The "Start dev proxy" seems already be covered since it's command is "npm run *:teamsfx" (see https://github.com/OfficeDev/teams-toolkit/blob/dev/packages/vscode-extension/src/debug/teamsfxTaskHandler.ts#L139)

For "Check dev proxy", how about adding more checks like:
- Task type is shell
- cwd is */proxy

Reply:
Checked that "Start dev proxy" does exist in current telemetry. So you may only need to add minimal scope for "Check dev proxy".

Reply:
Thanks. It is updated.